src/net_processing.cpp

```diff
@@ -2610,6 +2635,11 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             if (!MayHaveUsefulAddressDB(addr.nServices) && !HasAllDesirableServiceFlags(addr.nServices))
                 continue;
 
+            // Skip invalid addresses, maybe a new BIP155 network id from the future.
+            if (!addr.IsValid()) {
+                continue;
```

Comment:
Should skipping invalid BIP155 addresses be applied only for `addrv2` messages?
```cpp
if (!addr.IsValid() && msg_type == NetMsgType::ADDRv2) {
    continue;
}
```

Reply:
I realized that this code is not necessary so I dropped it altogether - an invalid address is not routable and is not reachable, so we are not going to do anything with it.